deploy.cfg.example

```diff
@@ -50,6 +50,21 @@ identity-providers = Globus, Google, OrcID
 # options will be provided to the identity provider as a set of keys and values where X is the key
 # any Y is the value.
 
+# identity-provider-<provider name>-envs - a comma separated list of names of alternate
```

Comment:
I realized several commits later that it's stupid to have per-provider environments. Locally I've changed it so that there's one `identity-provider-envs` key.